pkg/plugin/plugin.go

```diff
@@ -75,15 +75,15 @@ func ParseVersion(version string) (v Version, err error) {
 		return v, errors.New("plugin version is empty")
 	}
 
-	// A valid version string will have 4 submatches, each of which may be empty: the full string, "v",
-	// the integer, and the stage suffix. Invalid version strings do not have 4 submatches.
+	// A valid version string will have 3 submatches, each of which may be empty: the full string,
+	// the integer, and the stage suffix. Invalid version strings do not have 3 submatches.
 	submatches := verRe.FindStringSubmatch(version)
-	if len(submatches) != 4 {
+	if len(submatches) != 3 {
```

Comment:
Why the changes here would make it easier to understand?

Reply:
One of the submatches was `v`. We are not using `v` for anything so getting a submatch for it is inneficient. I just removed that submatch, and therefore I had to subtract 1 to the total length and the indexes.